footprints/main/models.py

```diff
@@ -994,8 +969,15 @@ def display_title(self):
             return self.title
 
     def owners(self):
-        role = Role.objects.get_owner_role()
-        return self.actor.filter(role=role)
+        return self.actor.filter(
+            role__name=Role.OWNER).select_related('person')
+
+    def owners_list(self):
+        a = ''
+        for actor in self.actor.all():
+            if actor.role.name == Role.OWNER:
+                a += actor.display_name() + ', '
+        return a[:-2]
```

Comment:
Oh, and it looks like the function above it already has the filter we want, so it could just be `actors = [actor.display_name() for actor in self.owners()]`